imagebuildah/stage_executor.go

```diff
@@ -590,10 +620,59 @@ func (s *StageExecutor) runStageMountPoints(mountList []string) (map[string]inte
 	return stageMountPoints, nil
 }
 
+func (s *StageExecutor) CreateNeededHeredocMountsForRun(files []imagebuilder.File) ([]Mount, error) {
```

Comment:
This method doesn't need to be exported.

Reply:
fixed, I'll squash dev commits when PR is approved.